pkg/tfsandbox/testdata/modules/test_module/outputs.tf

```diff
@@ -3,7 +3,7 @@ output "output1" {
 }
 
 output "sensitive_output" {
-  value     = terraform_data.example.output
+  value     = sensitive(terraform_data.example.output)
   sensitive = true
```

Comment:
Terraform CLI doesn't track this output as sensitive if the value itself isn't sensitive 🤔

Reply:
Tracking here https://github.com/pulumi/pulumi-terraform-module/issues/363 but it looks like this change can be reverted as soon as a new Terraform CLI is out